File: 200_code_python/13.py
# Viết bằng PYTHON
# Kiểm tra một số nhập vào từ bàn phím là số chẵn hay số lẻ.

# Nhập số từ bàn phím
a = int(input("Nhap mot so: "))

# Kiểm tra chẵn hay lẻ
if a % 2 == 0:
    print("So " + str(a) + " la so chan.")
else:
    print("So " + str(a) + " la so le.")

File: 200_code_cpp/13.cpp
// Viết bằng C++
// Kiểm tra một số nhập từ bàn phím là số chẵn hay số lẻ.

#include <iostream>
using namespace std;

int main() {
    // Khbao báo biến
    int a;

    // Nhập số từ bàn phím
    cout << "Nhap so: ";
    cin >> a;

    // Kiểm tra chẵn lẻ
    if (a % 2 == 0) {
        cout << "So " << a << " la so chan." << endl;
    } else {
        cout << "So " << a << " la so le." << endl;
    }

    return 0;
}